src/main/rx/rx.c

```diff
@@ -105,6 +105,14 @@ static uint8_t rcSampleIndex = 0;
 #define SERIALRX_PROVIDER 0
 #endif
 
+#ifndef SERIALRX_INVERTED
```

Comment:
We really have to stop adding new 'target custom default' defines. All of these will have to be undone as part of the work of moving to generalised targets.